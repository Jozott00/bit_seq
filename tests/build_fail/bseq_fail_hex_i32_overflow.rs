use bit_seq::bseq;

fn main() {
    let _ = bseq!(0xffffffff);
}