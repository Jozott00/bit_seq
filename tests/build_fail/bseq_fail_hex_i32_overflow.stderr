error: literal out of range for `i32`
 --> tests/build_fail/bseq_fail_hex_i32_overflow.rs:4:19
  |
4 |     let _ = bseq!(0xffffffff);
  |                   ^^^^^^^^^^
  |
  = note: the literal `0xffffffff` (decimal `4294967295`) does not fit into the type `i32` and will become `-1i32`
  = help: consider using the type `u32` instead
  = note: `#[deny(overflowing_literals)]` on by default
